src/services/command-factory-service.ts

```diff
@@ -30,7 +30,7 @@ const DEFAULT_COMMANDS: CommandsDefinition = {
   microservice: {
     'spring-boot': {
       clean: 'mvn clean',
-      build: 'mvn clean test',
+      build: 'mvn test',
```

Comment:
@zonia3000 should we do the same for package?

Reply:
I think that if we decide to remove the clean from build phase we can do the same for the package phase.
However I'm thinking that maybe we should invoke `mvn package` also in the build phase. Indeed, `mvn test` doesn't create the jar file and it would be a problem if a user executes `build` and then `pack` without the `-b` option.

Reply:
Update: we decided to remove the `-b` flag from `pack` command, so package phase always execute `mvn package` and it is correct to use `mvn test` here.